src/Storefront/Resources/views/storefront/page/product-detail/meta.html.twig

```diff
@@ -19,8 +19,13 @@
         <meta property="product:brand"
               content="{{ page.product.manufacturer.translated.name }}"/>
     {% endif %}
+
+    {% set metaPrice = page.product.calculatedPrice %}
+    {% if page.product.calculatedPrices.count > 0 %}
+        {% set metaPrice = page.product.calculatedPrices.last %}
+    {% endif %}
     <meta property="product:price:amount"
-          content="{{ page.product.calculatedPrice.unitPrice|round(context.currency.itemRounding.decimals) }}"/>
+          content="{{ metaPrice.unitPrice|round(context.currency.itemRounding.decimals) }}"/>
```

Comment:
```suggestion
          content="{{ product.calculatedCheapestPrice.unitPrice|round(context.currency.itemRounding.decimals) }}"/>
```

wouldn't it be better to use the cheapest price here? Or is there any reason to use the last calculated price?

Reply:
Sadly, not because cheapestPrice is over all other variants, not the current product.

Reply:
ah ok i see now that we have basically the same logic in `Resources/views/storefront/component/product/card/price-unit.html.twig`